.bash_profile

```diff
@@ -48,3 +48,19 @@ complete -W "NSGlobalDomain" defaults;
 
 # Add `killall` tab completion for common apps
 complete -o "nospace" -W "Contacts Calendar Dock Finder Mail Safari iTunes SystemUIServer Terminal Twitter" killall;
+
+# >>> conda initialize >>>
+# !! Contents within this block are managed by 'conda init' !!
+__conda_setup="$('/Users/raskolnikoff/anaconda3/bin/conda' 'shell.bash' 'hook' 2> /dev/null)"
```

Comment:
This part contains your very specific paths. I think you committed these by accident